docs/conf.py

```diff
@@ -16,13 +13,14 @@
 project = "gettsim"
 copyright = "2019, Hans-Martin von Gaudecker"
```

Comment:
Can we do something like "GETTSIM team" or is that legally a problem?

Reply:
I have no idea. But I have not fought any legal battles yet. I go with GETTSIM team.